checker/environ.ml

```diff
@@ -120,7 +120,7 @@ let anomaly s = anomaly (Pp.str s)
 let add_constant kn cs env =
   if Cmap_env.mem kn env.env_globals.env_constants then
     Printf.ksprintf anomaly ("Constant %s is already defined.")
-      (Constant.to_string kn);
+      (Constant.debug_to_string kn);
```

Comment:
This is not really debugging code, it is just error printing. What should it use instead of a debug printer?

Reply:
It is anomaly printing. I used `debug` as a continuation of existing terminology but we could use `internal`, or `unsafe`, or whatever other terminology.

Not that I'm a big fan of cryptic anomaly message. Users see some of them sometimes and they deserve an effort for understandable anomalies.

If we use the `Constant` printer, it is because we have nothing else at hand.